### PR TITLE
Bay Merge: The Hurt Me Good Edition

### LINKED ISSUE #159: Bay Ruined Xenos
Lets see:
Xenos can wear clothing
Xenos can pick up any item
Xenos can use melee weapons and guns
Xenos can be stunned
Xenos can use a lot of machines
Xeno leap stuns the Xeno just as long as the victim
Xenos can now gib people
Xenos can have brain damage and be effected by brain damage

In conclusion : http://puu.sh/d7QT7/f578ca3527.png
Bay ruined Xenos

### LINKED ISSUE #168: Mad Xenomorph Science with Knoxon
So, you want to do mad science involving xenomorph bodies recovered from the gateway? Great!
Today lets try and switch the brain of this dead xenomorph and this assistant brain!
Having trouble fitting it in the xeno's head because "Brains don't normally go there?", thats fine! Just shove it in the chest, works every time!
Tl;dr. You can transplant human brains into xenos, but not into their heads, into their chest (which works somehow), but unfortunately you can't close your new xenomorph.

### PR DESCRIPTION
**Pull request by Glloyd**
Ow

- [x] fix urist hardsuits
- [x] fix urist labcoats
- [x] fix species
- [x] borg sprites
- [x] update config
- [x] cry
- [x] revert heist removal
- [x] revert syndie removal
- [x] fix awaymaps
- [x] fix map
- [x] get things to compile
- [x] merge again

Fixes #159 
Fixes #168

### CODE REVIEW

**Comment by Glloyd** [comment]
FUCK

**Comment by Nienhaus** [comment]
Mess up?

**Comment by Glloyd** [comment]
We're good, there was some WIP shit in there from months ago, but I realized I had reverted it and the commits still showed.

**Comment by Nienhaus** [comment]
Ah

**Comment by TGameCo** [comment]
Is this the one with the paper update?

**Comment by Glloyd** [comment]
Yup, the tape's in there too for you.

**Comment by Glloyd** [comment]
I'll likely finish this on tuesday, I'm just working through map stuff before I go back and revert those specific commits.

**Comment by scrdest** [comment]
Working on the coats, uristclothes done.

**Comment by Glloyd** [comment]
Cheers, PR to my repo whenever you finish that.

**Comment by Glloyd** [comment]
All that's left is to fix the map, and we're golden.

**Comment by Nienhaus** [comment]
Whoo

**Comment by Glloyd** [comment]
Good news and bad news.

First off, it compiles. Yay.

Bad news is that it looks like this in DM: http://i.imgur.com/QBvW2Kn.jpg

That's because the map file got corrupted. It's okay though, and I didn't spend 3 hours with it for nothing. I have a plan. If it works, this'll be merged in the next half hour. If it doesn't, I'm gonna go die in a corner.

**Comment by scrdest** [comment]
Eh. Even if you didn't manage to fix it right now, we could always temporarily switch back to Baymap until it's fixed.

**Comment by Glloyd** [comment]
It worked. All is well.

Also, we've split too far that Baymap wouldn't work in an actual round. First thing that comes to mind is no room for blueshields, we've removed xenobiologists, botanists wouldn't spawn in, etc. etc.

**Comment by scrdest** [comment]
You might want to reintroduce Xenobiologist in the next update, as they get Xenozoology as they do now plus new Xenobotany.

**Comment by Glloyd** [comment]
There's added slots to scientist to compensate.

**Comment by Glloyd** [comment]
is this the part where I :+1: ?

**Comment by Glloyd** [comment]
:+1:

**Comment by Glloyd** [comment]
Honk.

**Comment by scrdest** [comment]
Don't :+1: yourself, you'll go blind. Here, let me help you: :+1:

**Comment by Glloyd** [comment]
It's over.

It's also 7:01 AM. What the actual fuck. I'm going to bed.

**Comment by Nienhaus** [comment]
>SLeep
